docs/release/trg-1/trg-1-07.md

```diff
@@ -0,0 +1,27 @@
+---
+title: TRG 1.07 - End User Manual
+sidebar_position: 1
+---
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Active | 22-Apr-2024 |                                                       |
+| Draft  | 21-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+End user manual is essential for providing user assistance and ensuring a positive user experience. The user of the application is the recipient of this documentation. Typical contents are description of functionality, steps that guide through critical processes and screenshots.
+
+## Description
+
+- The end user manual **must** exist if the product has a UI component
+- The end user manual **must** be stored in the folder: `/docs/user`
+  - It is up-to-date for every release
+  - English is a **must**
+- The `/docs/README.md` describes shortly what documentation is available
+
+## Best Practice
+
+- Favor diagrams as mermaid block in `.md` files directly over image files, where possible
+- Have a look to peer repositories within Tractus-X for inspiration
+- Anchor a link to your end user manual within your `README.md` (see [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1))
```

Comment:
I would add also two practices: role specifics and if you have an ui, make it available there (should be close to styleguide which has not yet been released)

```suggestion
- Anchor a link to your end user manual within your `README.md` (see [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1))
- Within the documentation consider role specifics of end users for better understanding
- If your application has an UI, make the documentation available within the document
```